lib/message.rb

```diff
@@ -1,6 +1,16 @@
 require 'pg'
 
 class Message
+
+  attr_reader :id, :username, :message, :time
+
+  def initialize(id:, username:, message:, time:)
```

Comment:
I liked the way you wrapped the peep into a message class :) Your initialize is clear :)